### ISSUE TITLE
riot-compiler@3.1.0 does not install with ubuntu 16.04 with -g flag (installing as a dependency of riot)

### ISSUE THREAD

**Issue by kgeorgy** [riot/compiler#98]
When running `sudo npm install -g riot` on ubuntu 16.04, there are some errors when installing the riot-compiler package (as dependency of riot).

It seems that the errors comes when npm runs post install scripts `npm run install-html && npm run install-js && npm run install-css`, specifically in `install-html` because it try to launch `npm i jade pug` without the `g` parameter, and also seems to launch them without the sudo command before. Why don't just put these packages as dependencies in the package.json file and let npm manage how to install them instead of installing them manually ? Then npm will install with the same environment (sudo and -g arg) the same way the as root package (in this case riot).

This is the complete console output

```
/usr/bin/riot -> /usr/lib/node_modules/riot/node_modules/riot-cli/lib/index.js

> riot-compiler@3.1.0 postinstall /usr/lib/node_modules/riot/node_modules/riot-compiler
> npm run install-html && npm run install-js && npm run install-css


> riot-compiler@3.1.0 install-html /usr/lib/node_modules/riot/node_modules/riot-compiler
> npm i jade pug

npm WARN deprecated jade@1.11.0: Jade has been renamed to pug, please install the latest version of pug instead of jade
npm WARN deprecated transformers@2.1.0: Deprecated, use jstransformer
npm WARN checkPermissions Missing write access to /usr/lib/node_modules
/usr/lib
├── jade@1.11.0 
└── pug@2.0.0-beta6 

npm ERR! Linux 4.4.0-21-generic
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "i" "jade" "pug"
npm ERR! node v5.12.0
npm ERR! npm  v3.8.6
npm ERR! path /usr/lib/node_modules
npm ERR! code EACCES
npm ERR! errno -13
npm ERR! syscall access

npm ERR! Error: EACCES: permission denied, access '/usr/lib/node_modules'
npm ERR!     at Error (native)
npm ERR!  { [Error: EACCES: permission denied, access '/usr/lib/node_modules']
npm ERR!   errno: -13,
npm ERR!   code: 'EACCES',
npm ERR!   syscall: 'access',
npm ERR!   path: '/usr/lib/node_modules' }
npm ERR! 
npm ERR! Please try running this command again as root/Administrator.

npm ERR! Please include the following file with any support request:
npm ERR!     /usr/lib/node_modules/riot/node_modules/riot-compiler/npm-debug.log

npm ERR! Linux 4.4.0-21-generic
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "run" "install-html"
npm ERR! node v5.12.0
npm ERR! npm  v3.8.6
npm ERR! code ELIFECYCLE
npm ERR! riot-compiler@3.1.0 install-html: `npm i jade pug`
npm ERR! Exit status 243
npm ERR! 
npm ERR! Failed at the riot-compiler@3.1.0 install-html script 'npm i jade pug'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the riot-compiler package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm i jade pug
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs riot-compiler
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls riot-compiler
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /usr/lib/node_modules/riot/node_modules/riot-compiler/npm-debug.log
/usr/lib
└── (empty)

npm WARN optional Skipping failed optional dependency /riot/chokidar/fsevents:
npm WARN notsup Not compatible with your operating system or architecture: fsevents@1.0.15
npm ERR! Linux 4.4.0-21-generic
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "install" "-g" "riot"
npm ERR! node v5.12.0
npm ERR! npm  v3.8.6
npm ERR! code ELIFECYCLE

npm ERR! riot-compiler@3.1.0 postinstall: `npm run install-html && npm run install-js && npm run install-css`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the riot-compiler@3.1.0 postinstall script 'npm run install-html && npm run install-js && npm run install-css'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the riot-compiler package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run install-html && npm run install-js && npm run install-css
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs riot-compiler
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls riot-compiler
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /usr/npm-debug.log
npm ERR! code 1

```

**Comment by cognitom** [comment]
Oh, that's my mistake #96.
@GianlucaGuarini I'll send a patch and re-publish soon.

**Comment by cognitom** [comment]
@kgeorgy really thank you for reporting this. I've fixed it and publish `riot-compiler@3.1.1` on npm.
https://www.npmjs.com/package/riot-compiler

Pls try it again :pray:

**Comment by kgeorgy** [comment]
@cognitom Now it is fine! (Wow, nice reactivity :) )
